python/lsst/ip/isr/isrMockLSST.py

```diff
@@ -322,7 +344,12 @@ def makeImage(self):
                 self.amplifierAddNoise(ampImageData, darkLevel, darkNoise, rng=rngDark)
 
             # 3. Add BF effect (electron) to imaging portion of the amp.
-            # TODO
+            if self.config.doAddSiliconPhysics is True:
+                self.amplifierAddSiliconPhysics(ampImageData,
+                                                self.config.rngSeed,
```

Comment:
So we don't want to use the seed directly if at all possible.  Instead we want to create the random number generator as above (see line 244), which in this case may be the galsim generator.  This ensures that we don't use the same seed for every amp (as is currently the case), but also ensures repeatability from run to run no matter what else is doing random numbers.